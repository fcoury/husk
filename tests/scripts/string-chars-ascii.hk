// Test string.chars() method - ASCII only

// Basic ASCII string
let hello = "hello";
let hello_chars = hello.chars();
println(hello_chars.len());
println(hello_chars[0]);
println(hello_chars[4]);

// Empty string
let empty = "";
let empty_chars = empty.chars();
println(empty_chars.len());

// String with spaces
let special = "a b c";
let special_chars = special.chars();
println(special_chars.len());
println(special_chars[1]);
println(special_chars[4]);