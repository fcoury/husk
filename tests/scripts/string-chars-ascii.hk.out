5
h
o
0
5
 
c
